index.js

```diff
@@ -90,14 +91,29 @@ async function main(opts) {
     }
   }
 
-  const base = process.env.BASE_URL || 'https://micro.readme.com';
+  // Get the base and remove the trailing slash that comes by default from vercel
+  const base = (process.env.BASE_URL || 'https://micro.readme.com').replace(/\/$/, '');
 
   return axios
     .post(`${base}/api/uploadSpec`, out, {
       headers: { 'X-API-KEY': options.key },
     })
-    .then(() => {
-      log('Successfully synced file to ReadMe Micro! 🦉');
+    .then(response => {
+      const body = response.data;
+      console.log('body', body);
```

Comment:
remove, or hide:

```suggestion
      core.debug('body', body);
```

or `core.isDebug()`, etc.

[`@actions/core` docs](https://github.com/actions/toolkit/tree/main/packages/core#logging)

<img width="276" alt="image" src="https://github.com/readmeio/readme-micro/assets/203725/6f76ee71-4f72-4699-80cc-ba63486556cc">